ignite/cmd/cmd.go

```diff
@@ -75,8 +76,14 @@ ignite scaffold chain github.com/username/mars`,
 	c.AddCommand(NewTools())
 	c.AddCommand(NewDocs())
 	c.AddCommand(NewVersion())
+	c.AddCommand(NewPlugin())
 	c.AddCommand(deprecated()...)
 
+	err := loadPlugins(c)
```

Comment:
Isn't it better to run it in the `PersistentPreRunE`?

Reply:
`PersistentPreRunE` is not executed when the user runs `ignite` w/o sub-commands, so the plugin won't be loaded in that case. 

Since the plugins add new commands, the load must also happen when the user runs only `ignite`, so he can see the added commands.